### PR TITLE
add an extra example of ResourcesTracker; change a misspelling of ResourcesTracker's comment

### PR DESCRIPTION
**Pull request by yangzhongke**
add an extra example of ResourcesTracker; change a misspelling of ResourcesTracker's comment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,6 +124,17 @@ using (ResourcesTracker t = new ResourcesTracker())
 	Mat mat4 = t.NewMat();
```
Comment (shimat) [inline README.md:124]:
```suggestion
	var mat4 = t.NewMat();
```

File: README.md [hunk 2]
```diff
@@ -124,6 +124,17 @@ using (ResourcesTracker t = new ResourcesTracker())
...
 	Cv2.Merge(new Mat[] { mats1[0], mats1[1], mats1[2] }, mat4);
 }
+
+using (ResourcesTracker t = new ResourcesTracker())
```
Comment (shimat) [inline README.md:128]:
```suggestion
using (var t = new ResourcesTracker())
```

**Review by yangzhongke — COMMENTED** [review]
cool. Agree!

**Comment by shimat** [comment]
Thank you :+1:

**Pull request merged**